pkg/userop/config.go

```diff
@@ -60,8 +61,8 @@ func (conf ClientConfig) validate() error {
 
 // GasConfig represents the configuration for the userop transaction gas fees.
 type GasConfig struct {
-	MaxPriorityFeePerGasMultiplier decimal.Decimal `yaml:"max_priority_fee_per_gas_multiplier" env:"MAX_PRIORITY_FEE_PER_GAS_MULTIPLIER"` // percentage
-	MaxFeePerGasMultiplier         decimal.Decimal `yaml:"max_fee_per_gas_multiplier" env:"MAX_FEE_PER_GAS_MULTIPLIER"`                   // percentage
+	MaxPriorityFeePerGasMultiplier decimal.Decimal `yaml:"max_priority_fee_per_gas_multiplier" env:"GAS_CONFIG_MAX_PRIORITY_FEE_PER_GAS_MULTIPLIER"` // percentage
+	MaxFeePerGasMultiplier         decimal.Decimal `yaml:"max_fee_per_gas_multiplier" env:"GAS_CONFIG_MAX_FEE_PER_GAS_MULTIPLIER"`                   // percentage
```

Comment:
Let's specify if the percentage is in range 0..1 or 0..100. In the latter case, we should also mention if decimals are allowed (like in `42.24`)

Reply:
Validation for these values in `validate` method of `ClientConfig` would be useful as well.

Reply:
1570e7574c29a4738557a0f53f6fb1a3df457291